pkg/apiserver/domain/service/helm.go

```diff
@@ -45,7 +45,7 @@ func NewHelmService() HelmService {
 type HelmService interface {
 	ListChartNames(ctx context.Context, url string, secretName string, skipCache bool) ([]string, error)
 	ListChartVersions(ctx context.Context, url string, chartName string, secretName string, skipCache bool) (repo.ChartVersions, error)
-	GetChartValues(ctx context.Context, url string, chartName string, version string, secretName string, skipCache bool) (map[string]interface{}, error)
+	GetChartValues(ctx context.Context, url string, chartName string, version string, secretName string, repoType string, skipCache bool) (map[string]string, error)
```

Comment:
Does the old API also call this function? The result changed, so, the old API was changed.

Reply:
Add old interface back.